site/docs/v1/tech/tutorials/switch-search-engines.adoc

```diff
@@ -0,0 +1,89 @@
+---
+layout: doc
+title: Switching search engines
+description: How to switch back and forth between the provided search engines
+---
+
+As of version 1.16, FusionAuth supports two search engines:
+
+* database
+* elasticsearch
+
+and you can easily switch between them. The strengths and limitations of each search engine type link:/docs/v1/tech/core-concepts/users#user-search[are documented here].
```

Comment:
Is this sentence supposed to span the bullet list? 

"supports two search engines: ....  and you can easily"